Makefile

```
@@ -634,6 +634,14 @@ CC_VER_MAJOR   := $(word 2,$(subst ., ,$(CC_INFO)))
 CC_VER_MINOR   := $(word 3,$(subst ., ,$(CC_INFO)))
 CC_VERSION     := $(CC_VER_MAJOR).$(CC_VER_MINOR)
 
+ifeq ($(call have_clang),y)
+ifeq ("$(ARCH)", "arm64")
+CFLAGS          += --target=aarch64-none-elf
+ASFLAGS         += --target=aarch64-none-elf
+CXXFLAGS        += --target=aarch64-none-elf
```

```suggestion
ARCHFLAGS             += --target=aarch64-none-elf
ISR_ARCHFLAGS         += --target=aarch64-none-elf
```
Would it make more sense to add those to `ARCHFLAGS`?